index.js

```diff
@@ -0,0 +1,46 @@
+const calculate = (array) => {
+  let newArray = []
+  const operators = ['+', '-', '*', '/']
+
+  for (let index = 0; index < array.length; index++) {
+
+    if (array[index] === undefined || array[index] === '') continue
+
+    if (typeof array[index] === 'number') {
+      newArray.push(array[index])
+      continue
+    }
```

Comment:
This can be removed because it !isNaN a number will return true for the data type number which you have on line 19.